### PR TITLE
security: Add an email template for announcing security fix of main branch

### PR DESCRIPTION
**Pull request by dio**
Commit Message: This proposes an email template for announcing the security fix of the main branch.

Risk Level: N/A
Testing: N/A
Docs Changes: This provides supporting information
Release Notes: N/A

### CODE REVIEW

File: security/email-templates.md [hunk 1]
```diff
@@ -167,3 +167,42 @@ Thanks,
...
+
+```
+Subject: Security fix of Envoy main branch (that includes $GITSHORTCOMMITHASH) is now available
+To: envoy-security-announce@googlegroups.com
+Cc: envoy-announce@googlegroups.com, envoy-security@googlegroups.com, envoy-maintainers@googlegroups.com
+
+Hello Envoy Community,
+
+The Envoy security team would like to announce the availability of the security fix for security
+defects introduced in the main branch by [$GITSHORTCOMMITHASH]($GITHUBCOMMITURL) commit. The defects
```
Comment (htuch) [inline security/email-templates.md:181]: "by the"

Comment (htuch) [inline security/email-templates.md:181]: "defect(s)" here and elsewhere.

File: security/email-templates.md [hunk 2]
```diff
@@ -167,3 +167,42 @@ Thanks,
...
+caused by [$GITSHORTCOMMITHASH]($GITHUBCOMMITURL) is not part of any Envoy stable releases.
```
Comment (htuch) [inline security/email-templates.md:182]: "are not part"

File: security/email-templates.md [hunk 3]
```diff
@@ -167,3 +167,42 @@ Thanks,
...
+
+$DEFECTSSUMMARY
+
+<!-- Provide details on features, extensions, configuration that make it likely that a system is
+vulnerable in practice. -->
+
+The CVSS score for this is [$CVSSSTRING]($CVSSURL).
+
+Including this [$FIXGITSHORTCOMMITHASH]($FIXGITHUBCOMMITURL) commit is encouraged to fix this issue.
```
Comment (htuch) [inline security/email-templates.md:191]: s/this/the/

File: security/email-templates.md [hunk 4]
```diff
@@ -167,3 +167,42 @@ Thanks,
...
+
+**Security fix timeline**
+
+1. The defects introduced in [$GITSHORTCOMMITHASH]($GITHUBCOMMITURL) was landed in the main branch
```
Comment (htuch) [inline security/email-templates.md:195]: "were landed"

Comment (htuch) [inline security/email-templates.md:195]: defect(s)

File: security/email-templates.md [hunk 5]
```diff
@@ -167,3 +167,42 @@ Thanks,
...
+   on $ORDINALDAY of $MONTH $YEAR at $PDTHOUR PDT ($GMTHOUR GMT).
+2. The fix [$FIXGITSHORTCOMMITHASH]($FIXGITHUBCOMMITURL) is merged into the main branch on
```
Comment (htuch) [inline security/email-templates.md:197]: "was merged"

File: security/email-templates.md (hunk 1 above)
Comment (attritionorg) [reply security/email-templates.md]: "of the fix for security defects" (security fix for security defects is redundant)

File: security/email-templates.md (hunk 3 above)
Comment (attritionorg) [reply security/email-templates.md]: Change CVSS to CVSSv2 or CVSSv3 depending which version you choose.

Comment (dio) [reply security/email-templates.md]: Hum, I need to defer to @envoyproxy/security-team for choosing the right version. I can do the find-and-replace, once it is decided.

Comment (htuch) [reply security/email-templates.md]: We use CVSSv3, but the actual CVSS string and URL contain this information, so I don't feel it's necessary to call that out.

**Pull request merged**